### ISSUE TITLE
Add support for --os

### ISSUE THREAD

**Issue by henryk-radoslaw-rychlik** [yuezk/GlobalProtect-openconnect#54]
Hello,

I work for a managed service provider and really like your software. Seems that some of our customers support windows clients only, which results in the following error message:
```
Matching client config not found
Creating SSL connection failed
Unknown error; exiting.
```

All that needs to be done to fix it is appending the following option:
```
--os=win
```

I already edited my local copy, would be great if you added support for it via command line arguments, or better, the GUI.

Thank you!

**Comment by Andreas85** [comment]
I think I'm having a similar problem. When I try to connect I get to the SAML window where I enter my username and password. But then I get this:

```
Microsoft
Sign in
Sorry, but we’re having trouble signing you in.

AADSTS50005: User tried to log in to a device from a platform (Unknown) that's currently not supported through Conditional Access policy. Supported device platforms are: iOS, Android, Mac, and Windows flavors.
```

@henryk-radoslaw-rychlik You said to simply add `--os=win`. Can you please explain where and how exactly?

Andreas

**Comment by dlenski** [comment]
> @henryk-radoslaw-rychlik You said to simply add --os=win. Can you please explain where and how exactly?

This option needs to be appended to the OpenConnect command-line, per the [OpenConnect man page](https://www.infradead.org/openconnect/manual.html).

https://github.com/yuezk/GlobalProtect-openconnect/blob/266ab658922b76844c0c8c58cde43d219e37da04/GPService/gpservice.cpp#L66-L70

**Comment by yuezk** [comment]
@Andreas85 Currently, the workaround is https://github.com/yuezk/GlobalProtect-openconnect/issues/42#issuecomment-816073455. The option could be added in the GUI in future release.

**Comment by yuezk** [comment]
It should have been resolved in 1.3.0, as described in https://github.com/yuezk/GlobalProtect-openconnect#passing-the-custom-parameters-to-openconnect-cli
